go/oprf/groups/group.go

```diff
@@ -95,6 +97,12 @@ func (c Ciphersuite) FromString(s string, pog PrimeOrderGroup) (Ciphersuite, err
 	case "SSWU-RO":
 		// do nothing
 		break
+	case "ICART-RO":
```

Comment:
Should be able to remove this once the P-384 string is corrected

Reply:
Solved ;)